index.js

```diff
@@ -181,4 +185,7 @@ class SlackNotifier extends NotificationBase {
     }
 }
 
-module.exports = SlackNotifier;
+module.exports = {
+    SlackNotifier: SlackNotifier,
+    validateConfigSlack: validateConfigSlack
```

Comment:
just call this `validateConfig`  context is already conveyed by the fact that it belongs to `notification-slack` package